docs/user/dashboard/dashboard.asciidoc

```diff
@@ -78,35 +81,74 @@ Begin with an empty dashboard, or open an existing dashboard.
 
 * To open an existing dashboard, click the dashboard *Title* you want to open.
 
+[float]
+[[add-panels]]
+=== Add panels
+
+To add panels to the dashboard, use one of the editors to create the panel, 
+add a panel from the *Visualize Library*, a table of live streaming logs, or the results from a machine learning anomaly detection job.
+
 [float]
 [[create-panels-with-lens]]
-=== Create panels
+==== Create panels
+
+To create panels, use one of the editors, then add the panel to the dashboard.
+
+. On the dashboard, click *Create panel*.
+
+. On the *New visualization* window, click the editor you want to use. *Lens* is recommended for most users.
+
+. To create the panel, configure the editor options.
+
+. To add the panel to the dashboard, choose one of the following options:
 
-Choose the type of panel you want to create, then save the panel to the dashboard.
+* To save the panel to the *Visualize Library*, click *Save to Library*, configure the options, then click *Save and return*.
++
+When panels are saved in the *Visualize Library*, the panel displays image:dashboard/images/visualize-library-icon.png[Visualize Library icon] in the header.
 
-. From the dashboard, choose one of the following options:
+* To add the panel to the dashboard without saving to the *Visualize Library*, click *Save and return*.
 
-* To create a panel, click *Create panel*, then click the panel type on the *New visualization* window. *Lens* is recommended for most users. 
+[float]
+[[add-panels-from-the-library]]
+==== Add panels from the library
 
-* To add a saved panel, click *Add from library*, then select the panel you want to add. When a panel contains a stored query, both queries are applied.
+Add panels that you've already created from the *Visualize Library*, 
 
-. To save the panel, click *Save* in the toolbar, then configure the *Save visualization* options.
+. On the dashboard, click *Add from library*.
 
-.. Enter the *Title* and optional *Description*.
+. On the *Add from library* flyout, click the panels you want to add to the dashboard.
 
-.. From the *Tags* drop down, select any applicable tags. 
+. To close the flyout, click *X*.
++
+When a panel contains a stored query, both queries are applied.
 
-.. Select *Add to Dashboard after saving*.
+. To unlink the panel from the *Visualize Library*, open the panel menu, then click *More > Unlink from library*.
```

Comment:
I feel like this step should be optional. Right now it sounds like a necessary step.

Reply:
I agree with Devon! Maybe there could be a line like "If you're wanting to make changes to the visualization without affecting it in other places, considering unlinking it from the Visualize Library" idk